__tests__/api.test.js

```diff
@@ -51,6 +51,32 @@ describe("GET /api/", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("should respond with a 200 status", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should respond with an object article containing the passed id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(body).toHaveProperty("article", expect.any(Object));
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+});
+
```

Comment:
You are missing some sad path tests for this endpoint: 

What happens if I make a request to /api/articles/notanid 

Would that be different to /api/articles/108489084 (doesn't exist) ?